client/src/components/uikit/Stack/index.tsx

```diff
@@ -35,24 +35,18 @@ const ownProps = ['axis', 'spacing', 'fluid', 'align', 'justify'];
 
 const vstack = (p: ThemedProps) => css`
   flex-direction: column;
-  & > *:not([data-spacer]) + *:not([data-spacer]) {
-    margin: ${getSpacing(p)} 0 0 0;
-  }
+  gap: ${getSpacing(p)};
 `;
 
 const fluid = (p: ThemedProps) => css`
   flex-wrap: wrap;
+  gap: ${getSpacing(p)};
```

Comment:
The fluid implementation can be simplified more. Gap is not needed here since it's defined in the vstack/hstack level. You can change the fluid stuff like this in the vstack and hstack:

```css
flex-wrap: ${p.$fluid ? 'wrap' : 'nowrap'};
```

Reply:
Actually now that I think about it we don't need vstack and hstack css helpers anymore since the gap usage is so simple that this suffices:

```css
flex-direction: ${p.$axis === 'x' ? 'row' : 'column'};
```